doc/security/specs/secure_boot/README.md

```diff
@@ -77,13 +77,31 @@ Devices produced before the key is known to be compromised are not protected by
 
 Additionally, each key is restricted to one of three "roles", which determine in which device states the key can be used.
 The roles are:
-* dev (development, only for devices in the `DEV` lifecycle state)
-* test (manufacturing and testing, only for devices in the `TEST_UNLOCK` lifecycle state)
-* prod (production, only for devices in the `PROD` or `PROD_END` lifecycle states)
+* `dev`: development, only for devices in the `DEV` lifecycle state
+* `test`: manufacturing and testing, only for devices in the `TEST_UNLOCK` lifecycle state
```

Comment:
nit: Only for devices in `TEST_UNLOCKED` and `RMA` lifecycle states.

Reply:
I have just updated the text and table, there was an error because it did not reflect the current code (which only allows `dev` keys in `DEV`). I believe #17364 was about allowing `dev` keys in `TEST` and `RMA`?